sky/skylet/job_lib.py

```diff
@@ -368,10 +368,15 @@ def get_statuses_payload(job_ids: List[Optional[int]]) -> str:
 
 def load_statuses_payload(
```

Comment:
Can we add some docstr for this function to list all the possible cases for the dict? For example:
{None: None} there is no job on the cluster
{1: RUNNING, 2: None} job 2 does not exist

Reply:
It is already included in the docstring of `core.py::job_status`. I added a reference to it 🫡